qiita_pet/handlers/study_handlers/metadata_summary_handlers.py

```diff
@@ -65,14 +65,14 @@ def get(self, arguments):
             # The prep template has been provided
             template = self._get_template(PrepTemplate, prep_template)
             back_button_path = (
-                "/study/description/%s?top_tab=prep_template_tab&sub_tab=%s"
-                % (study_id, template.id))
+                "%s/study/description/%s?top_tab=prep_template_tab&sub_tab=%s"
+                % (qiita_config.portal_dir, study_id, template.id))
```

Comment:
qiita_config is not imported in this file - I'm assuming that flake8 will complain